### DISCUSSION TITLE
combinatorialgames

### DISCUSSION THREAD

**Discussion by giscus** [jamesstephenbrown/NZSWebsite#25] [Announcements]
# combinatorialgames

A deep dive into Combinatorial Games, their history, and significance in mathematics and game theory.

https://nonzerosum.games/combinatorialgames.html

**Comment by andrewtaneglen** [comment]
This is an interesting game, as there would be many situations where the person before you would know they can't lose, but have the ability to let either you or the next person win the game (assuming, e.g. three players), so nearly all games would come down to a player being unable to win, but clearly able to determine the winner. This being the case, the best strategy would be to befriend the other players, such that they would be more inclined to give you the win when they can't win themselves.

**Reply by jamesstephenbrown** [reply]
True, given 2 players the first player to move can force a win, but as you say given 3 players, any 2 players working in tandem can win, even if they didn't start... Also, given 4 players, 2 cooperating players can secure a win, as long as they're not side by side - then if you have 2 such pairs cooperating, you have a game theory optimal trap again and then one player will win every time... I think, I'm going off some increasingly loose theoretical ping-pong now.
